certified-connectors/UiPath Orchestrator/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3420 @@
+            "type": "string",
+            "x-ms-summary": "Filter",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "in": "query",
+            "name": "$orderby",
+            "description": "Specifies the order in which items are returned. The maximum number of expressions is 5.",
+            "type": "string",
+            "x-ms-summary": "Order By",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "in": "query",
+            "name": "$top",
+            "description": "Limits the number of items returned from a collection. The maximum value is 1000.",
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "Top",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "in": "query",
+            "name": "$skip",
+            "description": "Excludes the specified number of items of the queried collection from the result.",
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "Skip",
+            "x-ms-visibility": "advanced"
+          }
+        ],
+        "produces":
+        [
+          "application/json"
+        ],
+        "responses":
+        {
+          "200":
+          {
+            "description": "Success",
+            "schema":
+            {
+              "$ref": "#/definitions/ODataValueOfIEnumerableOfFolderDto"
+            }
+          }
+        },
+        "summary": "Get Folders",
+        "tags":
+        [
+          "Folders"
+        ],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/odata/Jobs/UiPath.Server.Configuration.OData.StartJobs":
+    {
+      "post":
+      {
+        "consumes":
+        [
+          "application/json;odata.metadata=minimal;odata.streaming=true",
+          "application/json;odata.metadata=minimal;odata.streaming=false",
+          "application/json;odata.metadata=minimal",
+          "application/json;odata.metadata=full;odata.streaming=true",
+          "application/json;odata.metadata=full;odata.streaming=false",
+          "application/json;odata.metadata=full",
+          "application/json;odata.metadata=none;odata.streaming=true",
+          "application/json;odata.metadata=none;odata.streaming=false",
+          "application/json;odata.metadata=none",
+          "application/json;odata.streaming=true",
+          "application/json;odata.streaming=false",
+          "application/json"
+        ],
+        "description": "Creates a new job and sets it in Pending state for each robot based on the input parameters and notifies the respective robots about the pending job.",
+        "operationId": "Jobs_StartJobs",
+        "parameters":
+        [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "StartInfo - The information required to register the new jobs.",
+            "schema":
```

Comment:
You can use `definitions` for this schema as well.

Reply:
Ok, moved to definition.